tests/00-setup

```diff
@@ -36,9 +36,10 @@ createvenv() {
         [ $updated_downloadcache -eq 0 ] && bzr up $DOWNLOADCACHE
         virtualenv --distribute --extra-search-dir=$DOWNLOADCACHE $VENV
         . $ACTIVATE && \
-            yes w | env BZR_PLUGIN_PATH='-user' \
-            pip install --find-links releases --find-links jujugui-deps jujugui
-            pip install -r $TEST_REQUIREMENTS --find-links releases --find-links deps --find-links $DOWNLOADCACHE --no-allow-external --no-index
+        pip install --upgrade pip && \
```

Comment:
It is the pip in the venv not the system one.

The intent is to get a modern pip on trusty.  I guess I could only do it if the current is < 7.  

I'll see what that looks like.